### PR TITLE
Create 2024-05-21-5xx-error-messages.md

### PR DESCRIPTION
**Pull request by IngridKwok**
## Summary

Release note for the latest error messages. 

**Release**:
- [x] When ready
- [ ] After date: $DATE

### CODE REVIEW

**Comment by IngridKwok** [comment]
@rachelwhitton @stevector Please help merge. Shannon and Chris Yates already reviewed and approved.

**Comment by IngridKwok** [comment]
Image added. Ready to publish as reviews were done. @rachelwhitton @stevector

**Comment by rachelwhitton** [comment]
Assigning to @stevector for review

File: source/releasenotes/2024-05-21-5xx-error-messages.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Unbranded 5xx error messages launched
+published_date: "2024-05-21"
+categories: [general]
+---
+
+We’ve replaced all 5xx error messages with plain HTML and removed Pantheon branding for sites hosted on Pantheon.
```
Comment (stevector) [inline source/releasenotes/2024-05-21-5xx-error-messages.md:7]: [](url)
```suggestion
We’ve replaced with Pantheon branding on [all platform-generated 5xx error messages](guides/errors-and-server-responses/5xx-errors) with plain HTML. 
```

File: source/releasenotes/2024-05-21-5xx-error-messages.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+title: Unbranded 5xx error messages launched
+published_date: "2024-05-21"
+categories: [general]
+---
+
+We’ve replaced all 5xx error messages with plain HTML and removed Pantheon branding for sites hosted on Pantheon. 
+
+## Why this change
+
+Many of our customers, particularly those who are clients of our agency partners, may not be aware that their sites are hosted on Pantheon. For these clients, encountering Pantheon branded error messages could lead to confusion.
```
Comment (stevector) [inline source/releasenotes/2024-05-21-5xx-error-messages.md:11]:
```suggestion
Many of our customers, particularly those who are clients of our agency partners, may not be aware that their sites are hosted on Pantheon. For these clients, encountering Pantheon-branded error messages led to confusion.
```

**Review by stevector — CHANGES_REQUESTED** [review]
Some minor copy edits

File: source/releasenotes/2024-05-21-5xx-error-messages.md (hunk 1 above)
Comment (rachelwhitton) [reply source/releasenotes/2024-05-21-5xx-error-messages.md:7]: typo, needs a leading `/` for the link to work

Comment (stevector) [reply source/releasenotes/2024-05-21-5xx-error-messages.md:7]: fixed

File: source/releasenotes/2024-05-21-5xx-error-messages.md (hunk 2 above)
Comment (IngridKwok) [reply source/releasenotes/2024-05-21-5xx-error-messages.md:11]: Thanks for reviewing. Good to go. @stevector  @rachelwhitton

**Pull request merged**